src/design-system/assets-table/assets-table-token-profile.component.tsx

```diff
@@ -7,26 +7,48 @@ import { Text } from '../text';
 
 import * as cx from './assets-table-token-profile.css';
 
-interface Props {
-  imageSrc: string;
-  name: string;
-  description: string;
+const getImageAlt = ({
+  alt,
+  name,
+}: {
+  alt?: string;
+  name: string | ReactElement;
+}) => {
+  if (alt) return alt;
+  return typeof name === 'string' ? name : undefined;
```

Comment:
Should we have any fallback instead of `undefined`?

Reply:
I would prefer to avoid specifying not translated text.